README.md

```diff
@@ -1,82 +1,298 @@
+- **OpenTelemetry** - Distributed tracing and monitoring
 
-## CSS
-CSS is compiled using PostCSS and Tailwind. See configuration in `postcss.config.js` which imports packages defined in `package.json`.
+**Project Structure:**
+- `ManagementHub.Service/` - Main web API service
+  - `Areas/` - Feature areas organized by domain:
+    - `Debug/` - Debugging and development endpoints
+    - `Export/` - Data export functionality
+    - `Identity/` - Authentication and authorization
+    - `Languages/` - Language management
+    - `Ngbs/` - National Governing Body endpoints
+    - `Payments/` - Payment processing
+    - `Referees/` - Referee management
+    - `Tests/` - Test and certification endpoints
+    - `User/` - User management
+  - `Authorization/` - Authorization policies and handlers
+  - `Configuration/` - Application configuration
+  - `Jobs/` - Background job definitions
+  - `Swagger/` - API documentation configuration
+  - `Telemetry/` - Observability and monitoring
+- `ManagementHub.Models/` - Domain models and data structures
+- `ManagementHub.Storage/` - Data access layer and database context
+- `ManagementHub.Processing/` - Business logic and domain services
+- `ManagementHub.Serialization/` - JSON serialization configuration
+- `ManagementHub.Mailers/` - Email notification services
+- `ManagementHub.UnitTests/` - Unit test project
 
-Compiled CSS is included in the core application for single pages (like auth) and for the SPA.
+### Infrastructure
 
-We can use AWS CloudFront to be a CDN on top of our application. The service hosts static files such as CSS/JS in a normal way.
-We have a CF endpoint pointing at our service. We use the CF endpoint when referring to our static assets so that they're cached and geo distributed.
+**Docker:**
+- Multi-stage Docker builds for containerized deployment
+- Docker Compose configurations for local development with dependencies
+- Configurations available in `docker/` directory for dev, staging, and production environments
 
-In rails this was done automatically using these instructions https://devcenter.heroku.com/articles/using-amazon-cloudfront-cdn#adding-cloudfront-to-rails
-and by calling special tags in views.
+**CI/CD:**
+- GitHub Actions pipeline (`.github/workflows/pipeline.yml`)
+- Automated building, testing, linting, and security scanning
+- CodeQL security analysis
+- Docker image publishing to AWS ECR
+- Deployment to AWS Elastic Beanstalk (production)
+
+**Observability:**
+- OpenTelemetry instrumentation for distributed tracing
+- Support for Azure Monitor and OTLP exporters
+- Structured logging with request correlation
+
+## Building Instructions
+
+### Prerequisites
+
+- **.NET SDK 8.0** or later ([download](https://dotnet.microsoft.com/download))
+- **Node.js 18.x** or later ([download](https://nodejs.org/))
+- **Yarn** package manager ([install](https://yarnpkg.com/getting-started/install))
+- **PostgreSQL** database (for local development with real database)
```

Comment:
Let's make sure to note that postgresql and redis are optional and service can run with in memory database and cache.